stable_baselines/common/vec_env/vec_normalize.py

```diff
@@ -88,18 +131,47 @@ def reset(self):
         self.ret = np.zeros(self.num_envs)
         return self._normalize_observation(obs)
 
+    @staticmethod
+    def load(load_path, venv=None):
+        """
+        Loads a saved VecNormalize object.
+
+        :param load_path: the path to load from.
+        :param venv: the VecEnv to wrap.
+        :return: (VecNormalize)
+        """
+        with open(load_path, "rb") as file_handler:
+            vec_normalize = pickle.load(file_handler)
+        if venv is not None:
```

Comment:
Why would you allow to load a VecNormalize without an venv?
Also there is no check afterward for that...

Reply:
Good point. I was imagining sometimes wanting to load it just to get the normalization statistics, but I can't think of a use case other than debugging where you'd need this without an environment, so I've made the argument mandatory.